Source/monster.h

```diff
@@ -199,7 +209,7 @@ struct MonsterStruct { // note: missing field _mAFNum
 	uint16_t mMagicRes;
 	_speech_id mtalkmsg;
 	uint8_t leader;
-	MonsterRelation leaderflag;
+	MonsterRelation Relation;

```

Comment:
Wrong casing, but also this might be better names:
```suggestion
	LeaderRelation leaderRelation;
```

Reply:
Done 🙂 
I also thought about `PackRelation` but `LeaderRelation` is also good. 👍

Reply:
It relates back to the original name so might be easier for people hew about the old one :shrug: